modules/update-other-images.adoc

```diff
@@ -35,4 +35,8 @@ $ oc -n stackrox set image ds/collector compliance=registry.redhat.io/advanced-c
 ----
 $ oc -n stackrox set image ds/collector collector=registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:{collector-version} <1>
 ----
+Or if you're using collector slim:
+----
+$ oc -n stackrox set image ds/collector collector=registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:{collector-version} <1>
+----
 <1> If you use Kubernetes, enter `kubectl` instead of `oc`.
```

Comment:
Please duplicate this footnote at the command above which sets image for non-slim collector.